source/io/module_eggShell.js

```diff
@@ -134,6 +138,9 @@
         var Data_WriteDouble = Module.cwrap('Data_WriteDouble', 'void', ['number', 'number']);
         var EggShell_ExecuteSlices = Module.cwrap('EggShell_ExecuteSlices', 'number', ['number', 'number', 'number']);
         var Occurrence_Set = Module.cwrap('Occurrence_Set', 'void', ['number']);
+        var TypeRef_Name = Module.cwrap('TypeRef_Name', 'string', ['number']);
+        var TypeRef_SubElementCount = Module.cwrap('TypeRef_SubElementCount', 'number', ['number']);
+        var TypeRef_GetSubElementByIndex = Module.cwrap('TypeRef_GetSubElementByIndex', 'number', ['number']);
```

Comment:
If all the types are numbers you don't need to use Module.cwrap

Reply:
Agree, I unwrap them when I move them to the module.